docker/base-deps/Dockerfile

```diff
@@ -1,14 +1,30 @@
-# The deploy Docker image build a self-contained Ray instance suitable
-# for end users.
+# The base-deps Docker image installs main libraries needed to run Ray
```

Comment:
it looked like the old comment on this file had been copy-pasted from the `deploy` container

Reply:
Good catch.